lib/bump.rb

```diff
@@ -115,16 +129,24 @@ def commit_message(version, options)
         (options[:commit_message]) ? "v#{version} #{options[:commit_message]}" : "v#{version}"
       end
 
-      def commit(version, file, options)
-        return unless File.directory?(".git")
-        system("git add --update Gemfile.lock") if options[:bundle]
-        system("git add --update #{file} && git commit -m '#{commit_message(version, options)}'")
+      def commit(version, options)
+        return unless commit_requested?(options)
+        system("git commit -m '#{commit_message(version, options)}'")
         system("git tag -a -m 'Bump to v#{version}' v#{version}") if options[:tag]
       end
 
+      def git_add(file, options)
+        system("git add --update #{file}") if commit_requested?(options)
+      end
+
+      def commit_requested?(options)
+        options[:commit] && File.directory?(".git")
```

Comment:
... if commit was requested and git is not there that should blow up too, but for now keeping the old behavior ...